src/interop/op-supervisor/launcher.star

```diff
@@ -79,7 +79,7 @@ def launch(
         service,
     )
 
-    return "op_supervisor"
+    return struct(service=service)
```

Comment:
small suggestion: I think we should get into the habit of using "constructor" functions for the objects we return, so we can point to the constructor documentation wherever we consume the result

Reply:
how do you mean constructor functions? as in the `launch` function?